apple_bce.c

```diff
@@ -432,5 +432,7 @@ MODULE_LICENSE("GPL");
 MODULE_AUTHOR("MrARM");
 MODULE_DESCRIPTION("Apple BCE Driver");
 MODULE_VERSION("0.01");
+// ROLANDO ZAPPACOSTA: added this for it to get autoloaded when built as a module:
```

Comment:
```suggestion
// Autoload when built as a module:
```

`git blame` can show author of the line, no need to add it as a comment.

I'm not even sure whether such comment is necessary (it may be obvious), but it doesn't probably hurt.